src/renderer/views/Watch/Watch.js

```diff
@@ -304,6 +305,7 @@ export default defineComponent({
     this.activeFormat = this.defaultVideoFormat
 
     this.checkIfTimestamp()
+    this.currentPlaybackRate = parseFloat(this.$store.getters.getDefaultPlayback)
```

Comment:
The value is already a number (as `parseFloat` takes a string, this would convert it from a number to a string and then back to a number).
```suggestion
    this.currentPlaybackRate = this.$store.getters.getDefaultPlayback
```